preload

```diff
@@ -14,7 +14,6 @@ ${DIRECTORY}/data/settings
 ${DIRECTORY}/data/status
 ${DIRECTORY}/etc
 ${DIRECTORY}/icons/categories
-${DIRECTORY}/apps
```

Comment:
Why did you remove ${DIRECTORY}/apps from `mktimestamps`?
It would seem that this change would regenerate the list if a new app was added.

Reply:
Look up, it was in the list rwice